### DISCUSSION TITLE
How to add new miner models

### DISCUSSION THREAD

**Discussion by anselal** [anselal/antminer-monitor#197] [Q&A]
We need to update the config/models.json accordingly.
To find out the settings you can query your miner using AntminerMonitor in particular:

- Run the app
- Go to localhost:5000/MINER_IP/summary

It should give you all the info you need

**Comment by smwbev** [comment]
@anselal ))))
![Screenshot 2023-02-11 at 22 47 22](https://user-images.githubusercontent.com/413011/218275765-0a05f4a6-79ad-40ee-bc45-5c18bf632010.png)

**Reply by anselal** [reply]
So the miner has 360 chips in total?
In 4 boards? 
Then you should the chips like this:

`"chips": "72,72,72,72",`

**Reply by smwbev** [reply]
120 on 1 board 
3 boards

![Screenshot 2023-02-11 at 23 09 58](https://user-images.githubusercontent.com/413011/218276697-6c505ed7-3e83-4b58-b0ab-99d09893a17f.png)

**Reply by anselal** [reply]
Thnx. Will add it as a new model after I push the update with the new requirements

**Reply by smwbev** [reply]
> Thnx. Will add it as a new model after I push the update with the new requirements

What about graphs with temperatures and hash rate?
So that you can open each miner and look at its statistics (t+hr).
For an hour, a day, a week, a month, a quarter, half a year, a year )))

**Reply by smwbev** [reply]
full screenshot of the interface
![Screenshot 2023-02-12 at 00 21 30](https://user-images.githubusercontent.com/413011/218279595-a6dfb33d-0004-4256-b63b-57264513c02b.png)

**Reply by anselal** [reply]
I am not very creative with User Interfaces. I coded the backend on my own and did just a simple UI.
You are free to contribute if you want

**Reply by smwbev** [reply]
:)
![Screenshot 2023-02-12 at 01 27 57](https://user-images.githubusercontent.com/413011/218281887-b6865e20-147f-4348-ac94-b7a340915ddd.png)

**Reply by anselal** [reply]
@tehnoconductor you can connect the software to another service like Grafana and have all sort of customizable graphs there.
I did it for a customer once and it worked great.
